src/styles/calendar.scss

```diff
@@ -0,0 +1,34 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * #{$cell-size} + 10px);
+  gap: #{$border-width};
+  margin: 0 auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day {
+  width: #{$cell-size};
+  height: #{$cell-size};
+  border: #{$border-width} solid #000;
+  background-color: #eee;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)